### PR TITLE
Groovy Eclipse Formatter with Spotless

### PR DESCRIPTION
**Pull request by ham1**
I've tried to implement some more rules with Spotless i.e. `remove.unnecessary.semicolons`, however it doesn't seem to be able to parse the Spock tests. I've created this draft PR in case anyone has some time to look into it. I'm going to go back to JUnit5 migration.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -334,6 +335,8 @@ allprojects {
                     trimTrailingWhitespace()
                     indentWithSpaces(4)
                     endWithNewline()
+                    paddedCell() // Avoid cyclic ambiguities
+                    greclipse().configFile(grEclipseConfFile)
```
Comment (vlsi) [inline build.gradle.kts]: Is `paddedCell()` mandatory?
The documentation of Spotless says it should be used as a last resort only.

If we can't go without it I would refrain from adding `greclipse` altogether.

Comment (ham1) [reply build.gradle.kts]: It doesn't seem to work with or without it; so I can't say if it's required or not.

**Comment by vlsi** [comment]
It looks like `greclipse` does not handle Groovy+Spock files well.
I suggest we abandon the idea and format manually where required.

**Comment by vlsi** [comment]
Sample failure:

```
Step 'groovy eclipse formatter' found problem in 'src/components/src/test/groovy/org/apache/jmeter/control/ThroughputControllerSpec.groovy':
Multiple problems detected during step execution:
Snippet.groovy: 238: unexpected token: * @ line 238, column 1.
   * loops
```

Relevant source code (note `* loops` before `when`):

```groovy
    def "0 maxThroughput does not run any samplers inside the TC"() {
        given:
            sut.setStyle(ThroughputController.BYNUMBER)
            sut.setMaxThroughput(0)

            def loops = 3
            def loop = createLoopController(loops)

            def expectedNames = ["zero", "three"] * loops
        when:
            def actualNames = expectedNames.collect({
                loop.next().getName()
            })
        then:
            loop.next() == null
            actualNames == expectedNames
            sut.testEnded()
    }
```

**Pull request closed without merging**